setup.py

```diff
@@ -44,12 +44,14 @@
         'botocore==1.22.12',  # required by c7n 0.9.14
         'boto3==1.19.12',  # required by c7n 0.9.14
         'c7n==0.9.14',  # custodian
+        'elasticsearch==7.10.0',  # depend on elasticsearch server
+        'elasticsearch-dsl==7.4.0',
         'requests==2.27.1',  # rest api & lambda
         'PyGitHub==1.55',  # gitleaks
-        'elasticsearch==7.10.0',  # depend on elasticsearch server
-        "elasticsearch_dsl==7.4.0",
-        "sphinx==4.0.2",
-        "sphinx_rtd_theme==0.4.3",
+        'myst-parser==0.17.0',
+        'sphinx==4.5.0',
+        'sphinx-rtd-theme==1.0.0',
+
```

Comment:
Delete this enter and add the comment  '# readthedocs'
Also please put it according to a,b,c order